### ISSUE TITLE
[LISN / Python3] health/simple_status page not working

### ISSUE THREAD

**Issue by Didayolo** [codalab/codalab-competitions#3016]
The staff page https://codalab.lisn.upsaclay.fr/health/simple_status stopped working and now returns the following error:

```
The CodaLab site is currently unavailable.

Sorry, the page you are looking for is currently unavailable.

Please try again later.
```

**Comment by Didayolo** [comment]
It's now working again.

**Comment by Didayolo** [comment]
The issue is happening again. Here are the logs:

```Python
django            | 2021-11-04 17:12:48,585 ERROR Internal Server Error: /health/simple_status
django            | Traceback (most recent call last):
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 882, in _resolve_lookup
django            |     current = current[bit]
django            | TypeError: 'CompetitionSubmission' object is not subscriptable
django            | 
django            | During handling of the above exception, another exception occurred:
django            | 
django            | Traceback (most recent call last):
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/exception.py", line 41, in inner
django            |     response = get_response(request)
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 187, in _get_response
django            |     response = self.process_exception_by_middleware(e, request)
django            |   File "/usr/local/lib/python3.8/site-packages/django/core/handlers/base.py", line 185, in _get_response
django            |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
django            |     return view_func(request, *args, **kwargs)
django            |   File "/app/codalab/apps/health/views.py", line 127, in simple_health
django            |     return render(request, "health/simple_health.html", {
django            |   File "/usr/local/lib/python3.8/site-packages/django/shortcuts.py", line 30, in render
django            |     content = loader.render_to_string(template_name, context, request, using=using)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/loader.py", line 68, in render_to_string
django            |     return template.render(context, request)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/backends/django.py", line 66, in render
django            |     return self.template.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 207, in render
django            |     return self._render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 107, in instrumented_test_render
django            |     return self.nodelist.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 990, in render
django            |     bit = node.render_annotated(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 957, in render_annotated
django            |     return self.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 177, in render
django            |     return compiled_parent._render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/test/utils.py", line 107, in instrumented_test_render
django            |     return self.nodelist.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 990, in render
django            |     bit = node.render_annotated(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 957, in render_annotated
django            |     return self.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 72, in render
django            |     result = block.nodelist.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 990, in render
django            |     bit = node.render_annotated(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 957, in render_annotated
django            |     return self.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/loader_tags.py", line 72, in render
django            |     result = block.nodelist.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 990, in render
django            |     bit = node.render_annotated(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 957, in render_annotated
django            |     return self.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/defaulttags.py", line 216, in render
django            |     nodelist.append(node.render_annotated(context))
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 957, in render_annotated
django            |     return self.render(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 1040, in render
django            |     output = self.filter_expression.resolve(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 708, in resolve
django            |     obj = self.var.resolve(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 849, in resolve
django            |     value = self._resolve_lookup(context)
django            |   File "/usr/local/lib/python3.8/site-packages/django/template/base.py", line 890, in _resolve_lookup
django            |     current = getattr(current, bit)
django            |   File "/app/codalab/apps/web/models.py", line 1449, in size
django            |     size = get_submission_size(self) or 0
django            |   File "/app/codalab/apps/web/utils.py", line 211, in get_submission_size
django            |     total += get_filefield_size(submission, 'file', aws_attr='s3_file', s3direct=True)
django            |   File "/app/codalab/apps/web/utils.py", line 226, in get_filefield_size
django            |     size = obj_summary.size
django            |   File "/usr/local/lib/python3.8/site-packages/boto3/resources/factory.py", line 339, in property_loader
django            |     self.load()
django            |   File "/usr/local/lib/python3.8/site-packages/boto3/s3/inject.py", line 86, in object_summary_load
django            |     response = self.meta.client.head_object(
django            |   File "/usr/local/lib/python3.8/site-packages/botocore/client.py", line 337, in _api_call
django            |     return self._make_api_call(operation_name, kwargs)
django            |   File "/usr/local/lib/python3.8/site-packages/botocore/client.py", line 656, in _make_api_call
django            |     raise error_class(parsed_response, operation_name)
django            | botocore.exceptions.ClientError: An error occurred (404) when calling the HeadObject operation: Not Found

```

**Comment by Didayolo** [comment]
It seems that we have this error when we don't have the "error when saving edits" #3018

Maybe the fix for #3018 provokes this issue.

**Comment by Didayolo** [comment]
Once again, the page came back to life
